charts/kube-master/charts/etcd/templates/secrets.yaml

```diff
@@ -0,0 +1,15 @@
+{{/* vim: set filetype=gotexttmpl: */ -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "fullname" . }}
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+    release: {{ .Release.Name }}
+type: Opaque
+data:
+  openstack-auth-url: {{ required "missing openstack-auth-url" .Values.openstack.authURL | b64enc }}
+  openstack-username: {{ required "missing openstack-username" .Values.openstack.username | b64enc }}
```

Comment:
I think, we ultimately end up with a service user here. Does it have permissions to do the swift operations? I'm unsure. It might only have the `network_admin` role. Please check.

Reply:
It doesn't need to we create the swift container using the admin client und give the cluster service user access to only this swift container using swits acls